### ISSUE TITLE
Export Enumeration Field

### ISSUE THREAD

**Issue by ElenaRepository** [OrchardCMS/Orchard#8421]
Hi, I would like to share with you a question that I was asked when I tried to export a content type. I added an enumeration field to my content type, loaded values and set a default value for it.
Is it right that I do not export the value set as default when I export the content? 
It does not currently export the value parameter, unless you have manually published the content.

**Comment by sebastienros** [comment]
Correct, the default value is only assigned in the UI when an item is edited. If you don't save it from the UI then the default value is not set.
